satpy/readers/olci_nc.py

```diff
@@ -80,7 +80,6 @@ def __init__(self, filename, filename_info, filetype_info):
         self.nc = xr.open_dataset(self.filename,
                                   decode_cf=True,
                                   mask_and_scale=True,
-                                  engine='h5netcdf',
                                   chunks={'columns': CHUNK_SIZE,
```

Comment:
Is it possible to pass an `engine` keyword argument to the file handler to be used here? Eh or maybe it is better to leave it with xarray's default.

Reply:
I'd like to have the engine as an option, so if you just could have a quicklook to see if that's fixable, it would be great. Otherwise, we should find a way to have that set globally in the long term (maybe with donfig @djhoese ?)

Reply:
I've just added an `engine` option to the OLCI reader, which will allow the user to choose which engine they want.